### DISCUSSION TITLE
Can I trigger tekton pipeline with Github Actions

### DISCUSSION THREAD

**Discussion by emirot** [tektoncd/triggers#1589] [Q&A]
I ve only seen tekton being triggered by 
- lighthouse
-  prow
- via webhook push events

I would like to know if I could via github actions ?

**Comment by AlanGreene** [comment]
Short answer: yes 😸 

Can you share more information about your use case? Using Tekton Triggers you can create PipelineRuns in response to a number of events.

You can make a request to your EventListener from a job in GitHub Actions, e.g. using curl or any other method to make a HTTP request to the EventListener's defined endpoint.

Depending on your use case you could also respond to a variety of GitHub webhook events (not just `push`) using a [GitHub Interceptor](https://tekton.dev/docs/triggers/interceptors/#github-interceptors) to filter the events.

**Reply by emirot** [reply]
Hey @AlanGreene,

Thanks for your reply.
The use case would be to replace prow by using GitHub actions.
I haven't seen any example or anyone doing that so not sure if it's a good idea.
Would I miss green check marks in PR for example ?
But looks like there is a lot of integration that needs to be done as it has to be done for every pipeline?
